moveit2.repos

```diff
@@ -1,10 +1,10 @@
-repositories:
```

Comment:
I believe that removing the `repositories:` tag will break things. I'll run CI to verify it.